src/components/Pagination/Pager.js

```diff
@@ -0,0 +1,85 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import cx from 'classnames';
+import { noop } from '../../common/helpers';
+
+/**
+ * Pager component for Patternfly React
+ */
+const Pager = ({
+  baseClassName,
+  className,
+  messages,
+  disableNext,
+  onNextPage,
+  disablePrevious,
+  onPreviousPage
+}) => {
+  const classes = cx('pager', className);
+  return (
+    <ul className={classes}>
+      <li className={disablePrevious ? 'previous disabled' : 'previous'}>
+        <a
+          href="#"
+          className={disablePrevious ? 'disabled' : ''}
```

Comment:
we've been using `className={classNames({'disabled': disablePrevious})}` for this sort of thing